client/main.js

```diff
@@ -97,7 +97,11 @@ main = (function(main, global) {
     });
 
 
-    $('#nav-chat-btn').on('click', toggleMainChat);
+    $('#nav-chat-btn').on('click', function(event) {
+      if (!event.ctrlKey && !event.metaKey) {
```

Comment:
Agreed. You know I'll always prefer declarative coding. And it's worth considering it may not be both, but one at a time... so :+1: I'm in for that change